src/SFA.DAS.LevyTransferMatching.Web/Models/Pledges/RejectApplicationPostRequest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using SFA.DAS.Encoding;
+using SFA.DAS.LevyTransferMatching.Web.Attributes;
+using System.Collections.Generic;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Models.Pledges
+{
+    public class RejectApplicationPostRequest
```

Comment:
Name should be pluralised, as we are rejecting (potentially) multiple applications and this is a parameter to an action called `RejectApplications`

Reply:
I have pluralised it now. i.e. RejectApplicationsPostRequest